packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -39,22 +38,21 @@
 #endif
 
 namespace {
+constexpr uint256_t arbcore_schema_version = 1;
 constexpr auto log_inserted_key = std::array<char, 1>{-60};
 constexpr auto log_processed_key = std::array<char, 1>{-61};
 constexpr auto send_inserted_key = std::array<char, 1>{-62};
 constexpr auto send_processed_key = std::array<char, 1>{-63};
-constexpr auto logscursor_current_prefix = std::array<char, 1>{-66};
-
-constexpr auto sideload_cache_size = 20;
-constexpr uint256_t checkpoint_load_gas_cost = 1'000'000'000;
-constexpr uint256_t max_checkpoint_frequency = 1'000'000'000;
+constexpr auto schema_version_key = std::array<char, 1>{-64};
+constexpr auto logscursor_current_prefix = std::array<char, 1>{-120};
```

Comment:
Since the database was being redone anyways, I figured we might as well give more space to the *_key values so when we add more they don't get mixed in with _prefix